src/page-multi-facility/ScenarioSidebar.tsx

```diff
@@ -38,6 +40,11 @@ const promoTexts: { [promoType: string]: string } = {
     "Turn on 'Data Sharing' to provide your baseline data to public researchers, to help improve models of disease spread in prisons in the future.",
   addFacilities:
     "Add additional facilities to see the impact across your entire system.",
+  rtChart: `New! View the chart of Rt (the rate of spread over time) for any
```

Comment:
Ultra nit: to be cognitively consistent with the promo ordering logic, it would make sense to list this first in the map instead of last.